.github/workflows/build-toolchain.yml

```diff
@@ -54,8 +54,8 @@ on:
         default: false
         type: boolean
 
-      stable_point:
-        description: 'Use Repo Smart Sync'
+      build_compilers_with_llvm:
```

Comment:
I would rename this to `llvm_clang`. We could also invert this and change to `use_msvc`.